mlflow/utils/download_cloud_file_chunk.py

```diff
@@ -41,16 +41,26 @@ def main():
             download_path=args.download_path,
             http_uri=args.http_uri,
         )
+    except requests.exceptions.ChunkedEncodingError as e:
+        with open(args.temp_file, "w") as f:
+            json.dump(
+                {
+                    "retriable": True,
+                    "error": repr(e),
+                },
+                f,
+            )
+        raise
     except requests.HTTPError as e:
-        temp_file = args.temp_file
-        with open(temp_file, "w") as f:
+        with open(args.temp_file, "w") as f:
             json.dump(
                 {
-                    "error_status_code": e.response.status_code,
-                    "error_text": str(e),
+                    "retriable": e.response.status_code in (401, 403, 408),
+                    "error": repr(e),
                 },
                 f,
             )
+        raise
 
```

Comment:
Then should we catch all other Exceptions here and write to temp_file?

Reply:
We can, but that would remove stdout and stderr.

Reply:
I see, it's captured by _ChunkDownloadError